django/cantusdb_project/main_app/views/chant.py

```diff
@@ -128,228 +156,138 @@ def get_context_data(self, **kwargs):
-                else:
-                    chant_tally = f"<b>{cpl_count}</b> chants"
-                context[
-                    "concordances_summary"
-                ] += f"""
-                    <tr>
-                        <td>
-                            <a href="http://cantus.ispan.pl" target="_blank">Cantus Planus in Polonia</a> (CPL)
-                        </td>
-                        <td>
-                            <a href="http://cantus.ispan.pl/id/{chant.cantus_id}" target="_blank">{chant_tally}</a>
-                        </td>
-                    </tr>
-                """
-
-            # Hungarian Chant Database
-            hcd_count = len([True for c in concordance_chants if c["db"] == "HCD"])
-            if hcd_count:
-                if hcd_count == 1:
-                    chant_tally = "<b>1</b> chant"
-                else:
-                    chant_tally = f"<b>{hcd_count}</b> chants"
-                context[
-                    "concordances_summary"
-                ] += f"""
-                    <tr>
-                        <td>
-                            <a href="http://hun-chant.eu" target="_blank">Hungarian Chant Database</a> (HCD)
-                        </td>
-                        <td>
-                            <a href="http://hun-chant.eu/id/{chant.cantus_id}" target="_blank">{chant_tally}</a>
-                        </td>
-                    </tr>
-                """
-
-            # Slovak Early Music Database
-            csk_count = len([True for c in concordance_chants if c["db"] == "CSK"])
-            if csk_count:
-                if csk_count == 1:
-                    chant_tally = "<b>1</b> chant"
-                else:
-                    chant_tally = f"<b>{csk_count}</b> chants"
-                context[
-                    "concordances_summary"
-                ] += f"""
-                    <tr>
-                        <td>
-                            <a href="http://cantus.sk" target="_blank">Slovak Early Music Database</a> (CSK)
-                        </td>
-                        <td>
-                            <a href="http://cantus.sk/id/{chant.cantus_id}" target="_blank">{chant_tally}</a>
-                        </td>
-                    </tr>
-                """
+            ################################################
+            ### create context["concordances_databases"] ###
+            # (data to be unpacked in chant_detail.html to make concordances summary)
+            # i.e. this summary:
+            #   Displaying N concordances from the following databases:
+            #   Cantus Database (CD) - N chants
+            #   Fontes Cantus Bohemiae (FCB) - N chants
+            #   etc...
+            #   Gregorien.info - VIEW AT GREGORIEN.INFO
+            concordances_databases = [
+                {
+                    "name": "Cantus Database",
+                    "initialism": "CD",
+                    "base_url": "/",
```

Comment:
Use the `reverse()` method?

Reply:
I like this suggestion.

But, we don't have the homepage listed in any of our `urls.py`s. Several of us spent a while trying to figure out where the `/` path is being defined, plugging different plausible arguments (and lack of arguments) into `reverse()`, and our best guess is that the `/` path is maybe being defined/generated in the flatpages app.

If you know off the top of your head how to make this work, I'll happily make this change, but it's looking like more trouble than it's worth at the present moment.